src/rust/demikernel/config.rs

```diff
@@ -60,8 +60,15 @@ mod raw_socket_config {
     pub const SECTION_NAME: &str = "raw_socket";
     #[cfg(target_os = "linux")]
     pub const LOCAL_INTERFACE_NAME: &str = "linux_interface_name";
+
+    // the primary interface index. this should be the virtualized interface for VMs.
     #[cfg(target_os = "windows")]
     pub const LOCAL_INTERFACE_INDEX: &str = "xdp_interface_index";
+
+    // N.B. hyper-V VMs can have both NetVSC and VF interfaces working in tandem, in which case
+    // we need to listen to the corresponding VF interface as well.
+    #[cfg(target_os = "windows")]
+    pub const LOCAL_VF_INTERFACE_INDEX: &str = "xdp_vf_interface_index";
```

Comment:
Should this also be guarded behind "catpowder-libos" feature as well?

Reply:
The #[cfg...] does not apply to multiple lines in Rust. It's only applicable to the immediate next line.

Reply:
Makes sense.